convert_checkpoints.py

```diff
@@ -278,7 +278,7 @@ def _load_orig_llama_weight(input_ckpt_dir: epath.Path):
 
 def _load_hf_llama_weight(input_ckpt_dir: epath.Path):
   print(f"Loading checkpoint files from {input_ckpt_dir}.")
-  safetensors_files = input_ckpt_dir.glob("*.safetensors")
+  safetensors_files = list(input_ckpt_dir.glob("*.safetensors"))
```

Comment:
Is this breaking anything? It's only used in line 282 and 287, it runs on LLaMA weight for me. Gemma shouldn't call into this function

Reply:
yes. the input_ckpt_dir.glob("*.safetensors") returns generator, and on the line that checks size is going to exhaust it. So it would run but create a empty file.

Reply:
Ah...When I test locally I was using the same output folder..Thanks for fixing it..